doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -221,6 +221,7 @@ nRF9160 samples
 
       * Enabled building of bootloader FOTA update files.
       * Corrected handling of the bootloader FOTA updates.
+      * Enabled the at_host library to make it easier to update certificates.
```

Comment:
```suggestion
      * Enabled the :ref:`lib_at_host` library to make it easier to update certificates.
```

Reply:
OK, I made the change.  I added changes for two more samples, so please take a look at the latest.  If OK, please approve.